README.md

```diff
@@ -1,36 +1,62 @@
 # Tic Tac Toe Ruby Game (The Odin Project)
 
-> This project is part of the Microverse Ruby Curriculum and The Oding Project. 
+> This is a Tic Tac Toe game made with Ruby. It's a part of the Microverse Ruby Curriculum and The Odin Project. 
 
-## Built With
+## Built using
 
 - Ruby
+- Gitflow Workflow
+
+## UML Class diagram
+
+![screenshot](./UML_Class_diagram.png)
+
+## Project requirements
+- All user interaction and game flow should be kept only in main.rb
+- Use OOP
 
 ## Live Demo
 [Live preview:](https://repl.it/join/gjrzjjbm-marcoshdezcam1)
 
 ## Getting Started
 
-To run the game just open it with an IRB console or just run it. 
+### How to run the game?
+1 -> Clone this repo and open a terminal inside the folder where you extracted it.
+2 -> Run "ruby ./bin/main.rb" in your terminal
```

Comment:
- [ ] **OPTIONAL** To add a break between these lines, you can leave a blank line between them
- [ ] **OPTIONAL** Since you already made the file executable, you can leave the instructions simply as "Run `bin/main.rb`"
- [ ] **OPTIONAL** You can use backquotes (\`) to use formatting for inline code `just like this`

![image](https://user-images.githubusercontent.com/53256139/91685518-2e826600-eb20-11ea-956a-41eefe583bbd.png)